Source/UnitTest/cmake_core.cmake

```diff
@@ -122,18 +122,18 @@ elseif(${ISA_SIMD} MATCHES "avx2")
                 ASTCENC_F16C=1)
     endif()
 
-    target_compile_options(${ASTC_TEST}
+    target_compile_options(${ASTCENC_TEST}
         PRIVATE
             $<$<NOT:$<CXX_COMPILER_ID:MSVC>>:-mavx2 -mpopcnt -mf16c>
             $<$<CXX_COMPILER_ID:MSVC>:/arch:AVX2>)
 
 endif()
 
-target_link_libraries(${ASTC_TEST}
+target_link_libraries(${ASTCENC_TEST}
     PRIVATE
         gtest_main)
 
-add_test(NAME ${ASTC_TEST}
-         COMMAND ${ASTC_TEST})
+add_test(ASTCENC_TARGET_NAME ${ASTCENC_TEST}
```

Comment:
This is a mistake. "NAME" is keyword in "add_test" it shouldn't be renamed.

Reply:
Also how does the actual name shows up? looks like the "COMMAND" and "NAME" are the same.